test/class_test.ts

```diff
@@ -1952,4 +1952,20 @@ describe('classes', () => {
       })();
     `);
   });
+
+  it('produce a proper hack to trick Babel and into TypeScript that would be OK with UglifyJS -- and continues to produce results as would be expected', () => {
+      validateCS1(`
```

Comment:
Yep, this is the sort of test that I had in mind, thanks!

(This line is indented two much by two spaces, though, I think.)

Reply:
Yep, fixed.